main.go

```diff
@@ -16,6 +16,85 @@
 
 package main
 
+import (
+	"cmd-exclude-prefixes-k8s/internal/prefixcollector"
+	"cmd-exclude-prefixes-k8s/internal/utils"
+	"context"
+	"sync"
+
+	"github.com/networkservicemesh/sdk/pkg/networkservice/common/excludedprefixes"
+
+	"github.com/kelseyhightower/envconfig"
+	"github.com/sirupsen/logrus"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/networkservicemesh/sdk/pkg/tools/jaeger"
+	"github.com/networkservicemesh/sdk/pkg/tools/signalctx"
+	"github.com/networkservicemesh/sdk/pkg/tools/spanhelper"
+)
+
+const (
+	envPrefix = "exclude_prefixes_k8s"
+)
+
+// Config - configuration for cmd-exclude-prefixes-k8s
+type Config struct {
+	ExcludedPrefixes   []string `desc:"List of excluded prefixes" split_words:"true"`
+	ConfigMapNamespace string   `default:"default" desc:"Namespace of kubernetes config map" split_words:"true"`
+	ConfigMapName      string   `default:"nsm-config" desc:"Name of kubernetes config map" split_words:"true"`
```

Comment:
Should we pass output file here as well?

Reply:
Definitely

Reply:
Added PrefixesFilePath field.